Tests/js/views/assets/ez-usermenuview-tests.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-usermenuview-tests', function (Y) {
+    var renderTest, eventTest,
+        CLASS_HIDDEN = 'is-user-menu-hidden';
+
+    renderTest = new Y.Test.Case({
+        name: "eZ User Menu render test",
+
+        setUp: function () {
+            this.userMenuItemFireEventView = new Y.View();
+            this.view = new Y.eZ.UserMenuView({
+                items: [this.userMenuItemFireEventView]
```

Comment:
you should put at least 2 views and set on them a priority so that you can check that the views are sorted by priority.